### PR TITLE
ci: codecov update

### PR DESCRIPTION
**Pull request by insignias**
Issue #, if available:

Description of changes:

1. Github workflow update for Codecov
2. `Merge-release-to-develop-notification` to send out automatic notifications to create PRs to merge to develop. Automatic merge from `release` to `develop` disabled
3. README updates for Codecov badges for `main` and `develop` branch

Checklist:

* [ ] Have you successfully deployed to an AWS account with your changes?
* [ ] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/build-and-test.yml
Comment (DaveMadsen) [inline .github/workflows/build-and-test.yml:1]: does this file now need to be updated every time someone creates a new package within the repo?

File: codecov.yml [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+          - workbench-core-accounts
+      workbench-core-accounts-ui:
+        target: 90%
+        flags:
+          - workbench-core-accounts-ui
+      workbench-core-audit:
+        target: 90%
+        flags:
+          - workbench-core-audit
+      workbench-core-authentication:
```
Comment (DaveMadsen) [inline codecov.yml:32]: Authn/authz should have higher targets.

File: .github/workflows/build-and-test.yml
Comment (insignias) [reply .github/workflows/build-and-test.yml:1]: Unfortunately yes ! We need to create a flag for each package

File: codecov.yml
Comment (DaveMadsen) [inline codecov.yml:1]: Looks like another file which will need to be update every time a new package  is added.

**Review by DaveMadsen — DISMISSED** [review]

File: codecov.yml
Comment (insignias) [reply codecov.yml:1]: Yeah :(

File: workbench-core/user-management/README.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
-
 ⚠️ $\textcolor{red}{\text{Experimental}}$ ⚠️ : Not for use in any critical, production, or otherwise important deployments
 
-## Code Coverage
-| Statements                  | Branches                | Functions                 | Lines             |
-| --------------------------- | ----------------------- | ------------------------- | ----------------- |
-| ![Statements](https://img.shields.io/badge/statements-100%25-brightgreen.svg?style=flat) | ![Branches](https://img.shields.io/badge/branches-100%25-brightgreen.svg?style=flat) | ![Functions](https://img.shields.io/badge/functions-100%25-brightgreen.svg?style=flat) | ![Lines](https://img.shields.io/badge/lines-100%25-brightgreen.svg?style=flat) |
+# Workbench Core User Management
+## `main branch coverage`
+[![codecov](https://codecov.io/github/aws-solutions/solution-spark-on-aws/graph/badge.svg?flag=workbench-core-user-management)](https://app.codecov.io/github/aws-solutions/solution-spark-on-aws/tree/main)
```
Comment (htztomic) [inline workbench-core/user-management/README.md:5]:
```suggestion
[![codecov](https://codecov.io/github/aws-solutions/solution-spark-on-aws/branch/main/graph/badge.svg?flag=workbench-core-user-management)](https://app.codecov.io/github/aws-solutions/solution-spark-on-aws/tree/main)
```

**Review by htztomic — DISMISSED** [review]
nit: there are still remnants of istanbul-badges-readme in the browser-approved-package and autoinstallers that can be removed.

**Comment by htztomic** [comment]
main branch and develop branch code cov is not created. Will need to update those and they are not displaying on this branch's README

File: README.md [hunk 3]
```diff
@@ -1,21 +1,17 @@
...
-[![Cfn-nag-scan](https://github.com/aws-solutions/solution-spark-on-aws/workflows/Cfn-nag-scan/badge.svg)](https://github.com/aws-solutions/solution-spark-on-aws/workflows/Cfn-nag-scan/badge.svg)
-[![Secret-scan](https://github.com/aws-solutions/solution-spark-on-aws/workflows/Secret-scan/badge.svg)](https://github.com/aws-solutions/solution-spark-on-aws/workflows/Secret-scan/badge.svg)
-[![Lint-PR](https://github.com/aws-solutions/solution-spark-on-aws/workflows/Lint-pr/badge.svg)](https://github.com/aws-solutions/solution-spark-on-aws/workflows/Lint-pr/badge.svg)
-[![semantic-release: angular](https://img.shields.io/badge/semantic--release-angular-e10079?logo=semantic-release)](https://github.com/semantic-release/semantic-release)
-[![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)
+# Codecov
+## `main branch coverage`
+[![codecov](https://codecov.io/github/aws-solutions/solution-spark-on-aws/branch/main/graph/badge.svg?flag=root)](https://app.codecov.io/github/aws-solutions/solution-spark-on-aws/tree/main)
```
Comment (htztomic) [inline README.md:6]:
```suggestion
[![main-codecov](https://codecov.io/github/aws-solutions/solution-spark-on-aws/branch/main/graph/badge.svg?flag=root)](https://app.codecov.io/github/aws-solutions/solution-spark-on-aws/tree/main)
```
could we do something like this so we don't need add that label on top?

**Comment by insignias** [comment]
> main branch and develop branch code cov is not created. Will need to update those and they are not displaying on this branch's README

The codecov badges for `main` and `develop` should get created once the changes are pushed in to `main` and `develop`. After the badges are created they will start displaying in the README files

File: README.md (hunk 3 above)
Comment (insignias) [reply README.md:6]: There is no option to edit the codecov badge text as they are provided by codecov, that's why I had to add a label at the top

**Review by aws-tyler — DISMISSED** [review]
SWB related files look good

**Review by htztomic — APPROVED** [review]

**Review by htztomic — APPROVED** [review]

**Review by FernandoAranda — APPROVED** [review]

**Pull request merged**